### PR TITLE
Split changes.h

### PR DESCRIPTION
**Pull request by masterleinad**
In response to #3617.
Split `changes.h` up into individual contributions.

### CODE REVIEW

**Comment by davydden** [comment]
@dealii/dealii i suggest we don't merge anything before this PR is merged.

File: doc/news/changes/major/20160520Jean [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Improved: The step-44 tutorial now uses the new CellDataStorage class to
```
Comment (jppelteret) [inline doc/news/changes/major/20160520Jean]: I suspect this file should be named something other than ` doc/news/changes/major/20160520Jean`

File: doc/news/changes/major/20160430DenisDavydovJean [hunk 2]
```diff
@@ -0,0 +1,9 @@
+New: Add a collection of classes to manage user's quadrature point data:
```
Comment (jppelteret) [inline doc/news/changes/major/20160430DenisDavydovJean]: There seems to be a file naming issue here as well.

Comment (jppelteret) [inline doc/news/changes/major/20160430DenisDavydovJean]: Ditto with the other entries I've contributed to :-(

Comment (masterleinad) [inline doc/news/changes/major/20160430DenisDavydovJean]: Sorry, for that! I just didn't consider hyphens in authors' names.

**Comment by masterleinad** [comment]
@jppelteret Hyphens in author names should now be handled correctly.

File: doc/news/changes/minor/20160311TobyD [hunk 3]
```diff
@@ -0,0 +1,3 @@
+New: Added unit tests for complex-valued PETSc and SLEPc.
+<br>
+(Toby D. Young, Denis Davydov, 2016/03/11)
```
Comment (davydden) [inline doc/news/changes/minor/20160311TobyD]: @masterleinad another corner case with `.` in names (@oneliefleft ).

**Comment by jppelteret** [comment]
Awesome, thanks for making the change :-)

File: doc/news/changes/minor/20160311TobyD (hunk 3 above)
Comment (jppelteret) [inline doc/news/changes/minor/20160311TobyD]: I think that this is the last problematic case left - I couldn't spot any more issues or instances of people using special characters in their name.

Comment (davydden) [inline doc/news/changes/minor/20160311TobyD]: agree, i also checked the whole thing and did not notice any other issues.

**Comment by masterleinad** [comment]
@davydden Dots in author's names are now fine as well

**Pull request merged**